### ISSUE TITLE
vpp tracks missing sections `statseq`, `socksvr`  on  run vpp-agent-init using image vpp-agent:v2.3.0, vpp-agent:v2.3.1

### ISSUE THREAD

**Issue by denis-tingajkin** [ligato/vpp-agent#1543]
I just faced with a strange case. Please take a look

### Steps to reproduce:

1) Create Dockerfile (I've used name`Dockefile`)
```
FROM ligato/vpp-agent:v2.3.0 as runtime
RUN rm /opt/vpp-agent/dev/etcd.conf; 
```
Notice: reproduciable on versions 2.3.0+
2) Build image`docker build -t test:v1 .`
3) Run `docker run -ti test:v1 cat /etc/vpp/vpp.conf ` and make sure `statseg`, `socksvr` sections are presented in vpp.conf.
4) Run `docker run -ti test:v1 vpp-agent-init `

Actual: in logs contains
```
...
------------------------------------------------------------
 No socket file found at: /run/vpp/api.sock
 VPP binary API socket file is missing!

  - is VPP running with socket for binapi enabled?
  - is the correct socket name configured?

 To enable it add following section to your VPP config:
   socksvr {
     default
   }
------------------------------------------------------------
...
------------------------------------------------------------
 VPP stats socket file /run/vpp/stats.sock is missing!

  - is VPP running with stats segment enabled?
  - is the correct socket name configured?

 To enable it add following section to your VPP config:
   statseg {
     default
   }
------------------------------------------------------------
...
```
Expected: vpp doesn't track missing sections because of they passed via vpp.conf

**Comment by ondrej-fabry** [comment]
The message actually [comes from GoVPP](https://github.com/FDio/govpp/blob/master/adapter/statsclient/statsclient.go#L34) and it's just helpful notice, not any kind of "tracking" of configuration. You fail to connect, you get a notice that no socket was found at path you provided and then some pointers what might be wrong so you can go and check.

So, did you check the existence of the files?

```
 VPP stats socket file /run/vpp/stats.sock is missing!
```

```
 No socket file found at: /run/vpp/api.sock
```

**Comment by denis-tingajkin** [comment]
@ondrej-fabry I've executed on the running container from step 4 next commands:
input: `# ls /run/vpp/`
output:
input: `# find . -name "*.sock"`
output: ```./run/vpp-api.sock```

**Comment by ondrej-fabry** [comment]
I'm not sure what VPP version you are running. However you are obviously using wrong path for the binary API socket. The default path was changed in VPP from `/run/vpp-api.sock` to `/run/vpp/api.sock`, please update your `govpp.conf` accordingly to VPP you are running. 

Also it seems that your stats socket is not there either. There was a bug in VPP in older version that failed to create stats socket, because `/run/vpp` folder did not exist before starting VPP. As workaround, ensure this directory is created before starting VPP.

**Comment by denis-tingajkin** [comment]
> There was a bug in VPP in older version that failed to create stats socket if /run/vpp folder did not exist before starting VPP

@ondrej-fabry it is working for me :) 
I've added folder before vpp started and govpp.conf and it is worked 👍

**Comment by denis-tingajkin** [comment]
*Checking the issues about notificaiton/statspoller services*

**Comment by ondrej-fabry** [comment]
> Question: How can I change vpp version for image based on vpp-agent:2.3.0?

You can build it your self like this.

```
VPP_VERSION=2001 make images
```

**Comment by denis-tingajkin** [comment]
@ondrej-fabry probably you need on vpp-agent side add `/run/vpp/` this folder? Because this
reproducing on pure vpp-agent v2.3.0

**Comment by denis-tingajkin** [comment]
The problem should be solved in another issue https://github.com/ligato/vpp-agent/issues/1548
